### ISSUE TITLE
QGIS Python QgsLayoutExporter exportToImage produces no output

### ISSUE THREAD

**Issue by chleidersdorff** [qgis/QGIS#49436]
### What is the bug or the crash?

When trying to export a layout from python standalone exportToImage does not produce any output. I have tried witht the pdf option (exportToPdf) and it works as expected.

The below scrip used to work but one of the latest updates has broken the image layout export option.

The script runs if run through the qgis inbuilt python console, but does not work when run externally.

```#!/usr/bin/env python3
# import os
from qgis.core import (QgsProject, QgsLayoutExporter, QgsApplication)
# from osgeo import gdal # to supress errors
from datetime import datetime

# gdal.PushErrorHandler('CPLQuietErrorHandler') #disable error warnings

QgsApplication.setPrefixPath(r"C:\OSGeo4W\apps\qgis-ltr", True)

gui_flag = False
app = QgsApplication([], gui_flag)

app.initQgis()
project_path = 'Z:/01 GIS Admin/04 Workspaces/Cloncurry Exploration/CE_CCPRO_037/CE_CCPRO_037_ExplorationTenementStatus.qgz'

project_instance = QgsProject.instance()
project_instance.setFileName(project_path)
project_instance.read()

manager = QgsProject.instance().layoutManager()
# layout = manager.layoutByName("my_layout") # name of the layout
layout1 = manager.layouts()[0]  # first layout
layoutnam1 = manager.printLayouts()[0].name()  # Name of layout 1
layout2 = manager.layouts()[1]  # second layout
layoutnam2 = manager.printLayouts()[1].name()  # Name of layout 2

# Insert current date into name
today = datetime.now().strftime("%Y%m%d")
layoutnam1 = layoutnam1[:22]+today+layoutnam1[22:]
layoutnam2 = layoutnam2[:22]+today+layoutnam2[22:]

savepath = r"C:\TEMP"  # path to save the layouts

settings = QgsLayoutExporter.ImageExportSettings()
settings.dpi = 300
print(layoutnam1)
print(layoutnam2)
exporter = QgsLayoutExporter(layout1)
exporter.exportToImage(savepath +'\\'+ layoutnam1 + ".jpg", QgsLayoutExporter.ImageExportSettings())
exporter = QgsLayoutExporter(layout2)
exporter.exportToImage(savepath +'\\'+ layoutnam2 + ".jpg", QgsLayoutExporter.ImageExportSettings())

app.exitQgis()
```

### Steps to reproduce the issue

run script as filename.py in python 3 from external console from qgis.

### Versions

![image](https://user-images.githubusercontent.com/52182284/180137866-8627e09b-d3f9-4ef2-be54-0832c7eee39f.png)

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [X] I tried with a new QGIS profile

**Comment by agiudiceandrea** [comment]
The issue doesn't occur on my Windows 10 system with QGIS 3.26.1 (OSGeo4W).
@chleidersdorff, apart from the fact that it seems to me your code contains some weirdness, are you sure your project has 2 print layouts? The code is trying to export the first layout  to an image file whose name is taken from the second layout.

**Comment by chleidersdorff** [comment]
> The issue doesn't occur on my Windows 10 system with QGIS 3.26.1 (OSGeo4W).
> @chleidersdorff, apart from the fact that it seems to me your code contains some weirdness, are you sure your project has 2 print layouts? The code is trying to export the first layout to an image whose file name is taken from the name of a second layout.

Weird, I was playing with the code a bit so the reference is just a mistake. It's still runs for me in the qgis console. But not standalone.

**Comment by agiudiceandrea** [comment]
Could you please specify how did you install QGIS (using the OSGeo4W Standalone Installer, what exactly, or the OSGeo4W Network Installer), what is exactly the code to use to reproduce the issue (please style it with proper [Markup tags](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)) and how do you exactly run the code externally?

**Comment by chleidersdorff** [comment]
Hi @agiudiceandrea 

I installed though OSGEO4W, I have been running this script for a little while but it stopped working properly with one of the more recent updates. I have tried removing all the OSGEO installation and doing a fresh installation.

the scrip is simply run from pycharm or VS Code at the moment. I ahve set all the nessasry env varibles in windows to allow it to run standalone. However I can runn in inside the qgis console and it works and produces the jpg outputs however when i run it standalone it finishes withour errors but produces no jpg files. I have tried running it with the altered statements to output a pdf and it seems to work, but for what im doinf i really need the image option to work.
